pkg/kubernetes/views/deploymentconfig-panel.html

```diff
@@ -5,7 +5,7 @@
             <i class="fa fa-gear"></i>
         </div>
         <div class="listing-actions">
-            <button title="Delete" class="btn btn-danger btn-delete pficon pficon-delete" disabled></button>
+            <button id="delete-route" title="Delete" class="btn btn-danger btn-delete pficon pficon-delete delete-entity " data-toggle="modal" data-target="#delete-entity-dialog" data-link="{{ item.metadata.selfLink }}" data-key="{{item.key}}"></button>
```

Comment:
The id="xxxx" here is invalid. This is used multiple times on a page. It's not correct HTML to have multiple tags with the same id.